### ISSUE TITLE
Doubts about epoch settings

### ISSUE THREAD

**Issue by wincj** [mode-str/crossview#10]
Hello author, this is a very meaningful job
But I have two question
1. many cross perspective geolocation tasks have fixed epochs (such as 210 or 120 epochs, etc.) and use the weight obtained from the last epoch for testing. Is 200 epochs the best weight you have experimentally verified, or is it based on the experience of other papers?
2.  I don't seem to have seen any code about training the SUES-200 dataset in your code. Have you implemented it in the code or do I need to implement it myself?

I hope you can answer my question, because I think every round should be tested while training to know what the best result is. Do you think my opinion is flawed?
Thank you, I hope to receive your reply. Thanks

**Comment by mode-str** [comment]
1. After 200 epochs, our model converged. Based on our experience, we consider this to be the final result of our model.
2. The code for the train and test of the SUES-200 dataset and the code for the University-1652 dataset are the same except for the different file paths they read from. It is recommended that you adjust the organization of the images at each altitude of the SUES-200 dataset according to the structure of the University-1652 dataset and the README document.

**Comment by wincj** [comment]
Thank you for the author's prompt response. Thanks. But I need to rephrase my first question: I am curious if the weights you used for testing were determined based on minimizing the loss of the training set?
   Because I see that some jobs involve training and testing simultaneously, and then using the best performance on the test set as the final weight. But it seems that you used the weight with the minimum loss from the training set as the final test weight. Will the weight obtained in this way not perform the best on the test set, and may have better weight effects in other epoch's  checkpoints?
  Secondly, I found that when I used your code with convnext tiny, convnext small, and convnext base, the results were not significantly different. I believe you have also conducted experiments using a better backbone, and have you encountered the same situation as me? Do you think this is a reasonable phenomenon?
  I need to thank you again for your positive response and for your outstanding contribution!
